src/40_Dynamic_AMP/Dynamic_And_Cached_Data_Details.html

```diff
@@ -0,0 +1,58 @@
+<!-- ## Dynamic and cached product data details -->
```

Comment:
Just saw the name of this sample, I don't understand 'Data Details' in the title. Maybe: 'Mixing Dynamic and Cached Data'